floss/main.py

```diff
@@ -253,10 +254,34 @@ def make_parser(argv):
     logging_group.add_argument(
         "-q", "--quiet", action="store_true", help="disable all status output on STDOUT except fatal errors"
     )
+    logging_group.add_argument(
+        "--color",
+        type=str,
+        choices=("auto", "always", "never"),
+        default="auto",
+        help="enable ANSI color codes in results, default: only during interactive session",
+    )
 
     return parser
 
 
+def set_color_config(color):
```

Comment:
```suggestion
def set_color_config(color):
    sys.stdout.reconfigure(encoding='utf-8')
```

looks like we still need this to force the output encoding